package.json

```diff
@@ -21,14 +21,14 @@
     "@changesets/changelog-github": "^0.4.8",
     "@changesets/cli": "^2.26.0",
     "@manypkg/get-packages": "^1.1.3",
-    "@types/node": "^16.18.11",
+    "@types/node": "^20.11.16",
     "prettier": "^2.8.1",
     "resolve-from": "^5.0.0",
     "skuba": "5.1.0",
-    "tsup": "^6.5.0"
+    "tsup": "^8.0.1"
```

Comment:
Updated just to ensure `target: 'node20'` was valid. Only breaking changes were dropping support for older version of node.